hooks/pre-command

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+aws_account_id="697149045717"
+hash=$(echo -n "$(echo "${BUILDKITE_REPO}" | awk -F'[:.]' '{ printf $3 }')/${BUILDKITE_PIPELINE_SLUG}" | sha256sum | cut -c1-55)
+aws_role_arn="arn:aws:iam::${aws_account_id}:role/bk-${hash}-role"
+duration_seconds=${BUILDKITE_PLUGIN_OBLT_AWS_AUTH_DURATION:-3600}
```

Comment:
Can you add a README so it does say about the default values?

Reply:
https://github.com/elastic/oblt-aws-auth-buildkite-plugin/pull/1/commits/18a8a7773809e4985cc095096a817b33a3f4bd31